tests/AutoReview/FixerFactoryTest.php

```diff
@@ -619,7 +619,7 @@ public function testFixerPriorityComment(): void
         }
 
         if (0 === \count($fixersPhpDocIssues)) {
-            $this->addToAssertionCount(1);
+            $this->expectNotToPerformAssertions();
```

Comment:
this one was correct as well, the test build an array of possible issues, if there are no issues than increasing the assert count here is valid